keanu-python/tests/test_const.py

```diff
@@ -90,7 +90,7 @@ def test_const_does_not_take_empty_ndarray():
     with pytest.raises(ValueError) as excinfo:
         Const(ndarray)
 
-    assert str(excinfo.value) == "Cannot infer type because the ndarray is empty"
+    assert str(excinfo.value) == "Cannot infer type because array is empty"
```

Comment:
It doesn't throw a `ValueError` anymore does it? And if the array is empty, I guess a Java exception would be thrown?

Reply:
it still throws a different `ValueError` - from `__infer_class_from_array`: https://github.com/improbable-research/keanu/blob/48762e3c27463b2b4b0759e2c1cd7ebcb192fc88/keanu-python/keanu/context.py#L86